attributes/default.rb

```diff
@@ -9,7 +9,18 @@
 default['rabbitmq']['deb_package'] = "rabbitmq-server_#{node['rabbitmq']['version']}-1_all.deb"
 default['rabbitmq']['deb_package_url'] = "https://www.rabbitmq.com/releases/rabbitmq-server/v#{node['rabbitmq']['version']}/"
 
-default['rabbitmq']['rpm_package'] = "rabbitmq-server-#{node['rabbitmq']['version']}-1.noarch.rpm"
+case node['platform_family']
+when 'rhel', 'fedora'
+  case node['platform_version']
+  when < '6'
```

Comment:
I think this should be `<=` as currently version `6` will match the other branch.